resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -1,115 +1,407 @@
-                "ec2:GetConsoleOutput"
-            ],
-            "Resource": "*"
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "DescribeSnapShot",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeSnapshots",
+        "ec2:DescribeSnapshotAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:snapshot/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "CreateSnapShotsAllInstances",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateSnapshots"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "ManageSnapshot",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CopySnapshot",
+        "ec2:CreateSnapshot"
+      ],
+      "Resource": "arn:aws:ec2:*:*:snapshot/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "SendSerialConsoleSSHPublicKey",
+      "Effect": "Allow",
+      "Action": [
+        "ec2-instance-connect:SendSerialConsoleSSHPublicKey",
+        "ec2:GetSerialConsoleAccessStatus"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeNetworkInsightsPaths"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DeleteNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DeleteNetworkInsightsPath",
+        "ec2:StartNetworkInsightsAnalysis"
+      ],
+      "Resource": "arn:aws:ec2:*:*:network-insights-path/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:RequestTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "CreateNetworkInsightsPath",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateNetworkInsightsPath",
+        "ec2:DescribeNetworkInsightsAnalyses"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DeleteNetworkInsightsAnalysis",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DeleteNetworkInsightsAnalysis"
+      ],
+      "Resource": "arn:aws:ec2:*:*:network-insights-analysis/*"
+    },
+    {
+      "Sid": "DescribeClientVpn",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeClientVpnAuthorizationRules",
+        "ec2:DescribeClientVpnConnections",
+        "ec2:DescribeClientVpnRoutes",
+        "ec2:DescribeClientVpnTargetNetworks"
+      ],
+      "Resource": "arn:aws:ec2:*:*:client-vpn-endpoint/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
```

Comment:
Do these all have this tag?

Reply:
I think ClientVPC won't have red hat tag. Removing